recipes-extend/target-readme/files/images/Readme.md

```diff
@@ -0,0 +1,39 @@
+# RZ/G2L-SBC images folder
+
+## Description
+
+This directory contains various image files used for deploying and booting the RZ/G2L-SBC platform. These images include the core operating system images, Device Tree Blobs (DTBs), and any additional overlays required for the hardware configuration.
+
+## A top-level directory of images
+
+```
+images
+├── bl2_bp-rzg2l-sbc.bin
+├── bl2_bp-rzg2l-sbc.srec
+├── bl2-rzg2l-sbc.bin
+├── core-image-minimal-rzg2l-sbc.wic
+├── dtbs                                    <---- Directory containing Device Tree Blob files
+│   ├── overlays                                <---- Directory containing Device Tree Overlay files
+│   │   ├── rzg2l-sbc-can.dtbo
+│   │   ├── rzg2l-sbc-dsi.dtbo
+│   │   ├── rzg2l-sbc-ext-i2c.dtbo
+│   │   ├── rzg2l-sbc-ext-spi.dtbo
+│   │   ├── rzg2l-sbc-ov5640.dtbo 
```

Comment:
Please ensure you have checked the build output before making any changes to the hierarchy of README files, as rzg2l-sbc-*.dtbo does not exist.

Reply:
Update in the commit: fa4a70cd5eb6037a6490416b3944b4b8ce0fae77